pkg/policy/l4.go

```diff
@@ -121,6 +123,8 @@ func (l7 L7DataMap) GetRelevantRules(identity *identity.Identity) api.L7Rules {
 			if selector.Matches(identity.Labels.LabelArray()) {
 				rules.HTTP = append(rules.HTTP, endpointRules.HTTP...)
 				rules.Kafka = append(rules.Kafka, endpointRules.Kafka...)
+				rules.L7Proto = endpointRules.L7Proto // XXX: Need to check types are the same?
```

Comment:
The conflict management is done in `mergeL4IngressPort()`

Reply:
Right, removing the comment.